src/Execution/ContextFactory.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Nuwave\Lighthouse\Execution;
+
+use Illuminate\Http\Request;
+use Nuwave\Lighthouse\Schema\Context;
+use Nuwave\Lighthouse\Support\Contracts\CreatesContext;
+use Nuwave\Lighthouse\Support\Contracts\GraphQLContext;
+
+class ContextFactory implements CreatesContext
+{
+    /**
+     * Generate GraphQL context.
+     *
+     * @param Request $request
+     *
+     * @return GraphQLContext
+     */
+    public function generate(Request $request): GraphQLContext
+    {
+        $user = app()->bound('auth') ? auth()->user() : null;
```

Comment:
Just recently came across something in the Laravel docs - we do not have to fetch the user, it is present as `$request->user()` https://laravel.com/docs/5.7/authentication#retrieving-the-authenticated-user